rubin_scheduler/scheduler/detailers/detailer.py

```diff
@@ -328,6 +330,54 @@ def __call__(self, observation_list, conditions):
         return observation_list
 
 
+class StartFieldSequenceDetailer(BaseDetailer):
+    """Append a sequence of observations to the start of a list
+
+    Parameters
+    ----------
+    ang_distance_match : `float`
+        How close should an observation be on the sky to be considered
+        matching (degrees).
+    time_match_hours : `float`
+        How close in time to demand an observation be matching (hours).
+    """
```

Comment:
science_program : `str` or None
     The science program to match.  None will ignore science_program.
scheduler_note : `str` or None
     The scheduler note to match.  None will ignore scheduler_note.
sequence_obs_list : `list` [`ObservationArray`]
     The observations to prepend. Set the science_program appropriately for these additional visits.